### DISCUSSION TITLE
👋 Is it possible to default the control panels site based on the domain it is logged in from?

### DISCUSSION THREAD

**Discussion by jamie-brown** [statamic/cms#8654] [Q&A]
Hey everyone, 

So I was wondering if there was already a Statamic way to default what site is set in the control panel based on the URL I'm accessing it from?

For example, if I'm using multisite and have these 3 domains:

```
- https://website-one.com/ (default)
- https://website-two.com/
- https://website-three.com/
```

If I access the control panel from `https://website-three.com/cp`, can the site within the control panel default to `website-three`?

The current behaviour is that `website-one` is the default site that shows no matter the URL the control panel is being accessed from.

I hope I have explained myself properly.

Thanks all.
Jamie

**Comment by duncanmcclean** [comment]
I don't believe this is currently possible.

However, it is related to statamic/ideas#737 and #3796.
